Error: UNREACHABLE
jp2/jp2_cod.c:304: unreachable: This code cannot be reached: "abort();".

jp2/jp2_cod.c:514: unreachable: This code cannot be reached: "abort();".

jp2/jp2_enc.c:354: unreachable: This code cannot be reached: "abort();".

diff -up jasper-1.900.1/src/libjasper/jp2/jp2_cod.c.unreachable jasper-1.900.1/src/libjasper/jp2/jp2_cod.c
--- jasper-1.900.1/src/libjasper/jp2/jp2_cod.c.unreachable	2007-01-19 22:43:05.000000000 +0100
+++ jasper-1.900.1/src/libjasper/jp2/jp2_cod.c	2011-06-27 15:28:13.083137952 +0200
@@ -301,7 +301,6 @@ jp2_box_t *jp2_box_get(jas_stream_t *in)
 	}
 
 	return box;
-	abort();
 
 error:
 	if (box) {
@@ -511,7 +510,6 @@ int jp2_box_put(jp2_box_t *box, jas_stre
 	}
 
 	return 0;
-	abort();
 
 error:
 
diff -up jasper-1.900.1/src/libjasper/jp2/jp2_enc.c.unreachable jasper-1.900.1/src/libjasper/jp2/jp2_enc.c
--- jasper-1.900.1/src/libjasper/jp2/jp2_enc.c.unreachable	2007-01-19 22:43:05.000000000 +0100
+++ jasper-1.900.1/src/libjasper/jp2/jp2_enc.c	2011-06-27 15:27:58.858353979 +0200
@@ -351,7 +351,6 @@ int sgnd;
 	}
 
 	return 0;
-	abort();
 
 error:
 
